client/src/hooks/useSales.jsx

```diff
@@ -81,19 +97,35 @@ const useSales = (sale) => {
   const handleSubmit = async () => {
     setIsLoadingButton(true);
     try {
-      await createSale({
+      const { data: saleInfo } = await createSale({
         bikeId: selectedBikeId,
         customerEmail: emailRef.current.value,
         customerName: name,
         quantity: quantityRef.current.value,
       });
-      await createShipment({
+      const { data: shpmtInfo } = await createShipment({
         company: name,
         location: location,
         status: 'Ordered',
         deliveryDate: deliveryDate,
         shippingDate: shippingDate,
       });
+      await sendEmail({
+        to: [userStore.email, saleInfo.customerEmail],
+        subject: `ERP Sales: Sale ${saleInfo._id} Placed`,
+        emailBody: `Information about the sale\n\nSale Id: ${saleInfo._id} \nCustomer: ${saleInfo.customerName} \nBike Id: ${saleInfo.bikeId} \nQuantity: ${saleInfo.quantity} \nSale Status: ${saleInfo.status}`,
+      });
+      await sendEmail({
```

Comment:
Dont change this now, but in future when we have a bunch of independent requests one after the other,
you can use await Promise.all(sendEmail, sendEmail, createThis, createThat) to run all the async calls in parallel to cut load time

Reply:
the speed increase isnt always noticeable especially since our requests are small and fast

Reply:
good call!

Reply:
Agree!